src/components/Home/CodeBlocks/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { InternalMenu } from 'components/MainNav'
+import { docsMenu } from '../../../navs'
+import React, { useState } from 'react'
+import ContentViewer from 'components/ContentViewer'
+import productOSContent from './ProductOS'
+import productAnalyticsContent from './ProductAnalytics'
+import sessionReplayContent from './SessionReplay'
+import featureFlagsContent from './FeatureFlags'
+import abTestingContent from './ABTesting'
+
+const content = [productOSContent, productAnalyticsContent, sessionReplayContent, featureFlagsContent, abTestingContent]
+
+export default function CodeBlocks() {
+    const [activeIndex, setActiveIndex] = useState(0)
+    return (
+        <div className="max-w-screen-2xl mx-auto relative pb-12">
+            <h2 className="text-4xl lg:text-6xl text-center mb-1">Code-based</h2>
```

Comment:
```suggestion
            <h2 className="text-4xl lg:text-6xl text-center mb-1">Mmmn, code examples</h2>
```

Reply:
thought process:

* this is quite funny but I think it is potentially confusing (does this product have a ux) if taken seriously
* hence this is more obviously funny, _sans_ confusion